haystack/document_stores/deepsetcloud.py

```diff
@@ -9,7 +9,7 @@
 from haystack.utils import DeepsetCloud
 
 
-DEFAULT_API_ENDPOINT = f"DC_API_PLACEHOLDER/v1"  # TODO
+DEFAULT_API_ENDPOINT = "https://api.cloud.deepset.ai/api/v1"
```

Comment:
Looks like this is superfluous. The actual `DEFAULT_API_ENDPOINT` has moved to haystack/utils/deepsetcloud.py. There it makes sense to adjust it. But this one can be safely removed.